client/src/util/formatDate.ts

```
@@ -1,14 +1,14 @@
 /* eslint-disable no-nonstandard-date-formatting/no-nonstandard-date-formatting */
 import { format } from "date-fns";
 
-export function formatDate(date: Date): string {
-  return format(date, "MM/dd/yyyy");
+export function formatDate(date: Date | null): string {
```

I do not think this should take a possible null. It creates a large surface area for silent errors in getting dates back from the server.